src/sonic-frr/patch/0050-bgpd-backpressure-Avoid-use-after-free.patch

```diff
@@ -27,12 +34,12 @@ index b1f8f19594..bb3cd62950 100644
 -			zebra_announce_del(&bm->zebra_announce_head, dest);
  		}
  	}
- 
+ 	bgp_evpn_remote_ip_hash_destroy(vpn);
```

Comment:
Why is this change introduced?

Reply:
The patch was failing to be applied in its current shape. So, I had to manually apply the existing patch and recreate the patch file backwards.
Note that bgp_evpn_remote_ip_hash_destroy(vpn); is an existing code. It has not been added by me. This line has been autogenerated during patch creation by git diff command,